src/pages/index.js

```diff
@@ -127,19 +259,38 @@ profileEditButton.addEventListener("click", () => {
 
 /////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
+avatarEditButton.addEventListener("click", () => handleAvatarModalOpen());
+avatarModalCloseButton.addEventListener("click", () =>
+  handleAvatarModalClose()
+);
+avatarModalSubmitButton.addEventListener("submit", (event) => {
```

Comment:
Please, delete it

Please, create an avatar popup using `PopupWithForm`


Closing, opening, submit handler - everything is done by `PopupWithForm`